docs/conf.py

```diff
@@ -51,7 +51,13 @@
     # 'ipython_console_highlighting',
     'scanpydoc',
     "sphinx_search.extension",
-    *[p.stem for p in (HERE / 'extensions').glob('*.py')],
+    # Locally defined
+    "debug_docstrings",
+    "github_links",
+    "param_police",
+    "typed_returns",
+    # "function_images",
+    # *[p.stem for p in (HERE / 'extensions').glob('*.py')],
```

Comment:
Yes, this should be generated with some plotting gallery plugin instead.

Let’s move this part to the PR that adds this plugin.

Reply:
As mentioned in the main PR comment, I don't think there should be example plots in the table of contents for the reference docs. Examples fit in tutorials, the docs for the function, and examples, but not the table of contents.